pkgs/development/python-modules/dagger-io/default.nix

```diff
@@ -0,0 +1,60 @@
+# pkgs = import (fetchTarball {
+# 	url = "https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.tar.gz";
+# 	sha256 = "sha256:0s4hvb6zgbi3nqnh2r4q129q4672fv65cy02mxchddz7p5hg5sma";
+# }) {};
+
+{ buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, anyio
+, attrs
+, cattrs
+, graphql-core
+, gql
+, httpx
+, beartype
+, platformdirs
+, typing-extensions
+, rich
+, typer
+, strawberry-graphql
+, shellingham
+, colorama
+,
+}:
+
+buildPythonPackage rec {
+  pname = "dagger-io";
+  version = "v0.6.1";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub
+    {
+      owner = "dagger";
+      repo = "dagger";
+      rev = version;
+      sha256 = "sha256-9QQ6aDCkTWNq5KOSGF6FH6UQrOYa51ctW3CMcGrCJAQ=";
+    } + "/sdk/python";
```

Comment:
We can use `sourceRoot` here.

Reply:
Sorry Im kinda new on NixOS. Could you provide an example or documentation of how to use sourceRoot? I was looking for info and just got working with this.

Reply:
Sure. What about the [unpackPhase section of the nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#ssec-unpack-phase)?
Examples can be found by running `grep sourceRoot pkgs/development/python-modules/*/*.nix`.

Reply:
Ok, Ill take a look, thank you so much for your time, is really helpfull.